Resources/Locale/en-US/escape-menu/ui/options-menu.ftl

```diff
@@ -102,6 +102,8 @@ ui-options-vp-vertical-fit-tooltip = When enabled, the main viewport will ignore
 ui-options-vp-low-res = Low-resolution viewport
 ui-options-parallax-low-quality = Low-quality Parallax (background)
 ui-options-ambient-occlusion = Show Ambient Occlusion
+ui-options-heat-distortion = Heat distortion
+ui-options-heat-distortion-tooltip = Enables a shader for hot atmos gases.
```

Comment:
You shouldn't need to acknowledge what a "shader" is. This should say what it does (player visible) and that it's more performance intensive.